acct.bionimbus.org/portal/gitops.json

```diff
@@ -79,253 +93,329 @@
         {
           "icon": "dictionary",
           "link": "/DD",
-          "color": "#a2a2a2",
+          "color": "#1a3057",
           "name": "Dictionary"
         },
         {
           "icon": "exploration",
-          "link": "/files",
-          "color": "#a2a2a2",
+          "link": "/explorer",
+          "color": "#1a3057",
           "name": "Exploration"
         },
         {
           "icon": "query",
           "link": "/query",
-          "color": "#a2a2a2",
+          "color": "#1a3057",
           "name": "query"
         },
         {
           "icon": "workspace",
-          "link": "/workspace",
-          "color": "#a2a2a2",
+          "link": "#hostname#workspace/",
+          "color": "#1a3057",
           "name": "Workspace"
         },
         {
           "icon": "profile",
           "link": "/identity",
-          "color": "#a2a2a2",
+          "color": "#1a3057",
           "name": "Profile"
         }
       ]
     },
-    "topBar": {
-      "items": [
-        {
-          "icon": "upload",
-          "link": "/submission",
-          "name": "Browse Data"
-        },
-        {
-          "link": "https://gen3.org/resources/user/",
-          "name": "Gen3 Documentation"
-        }
-      ]
-    },
     "login": {
-      "title": "ACCOuNT Data Commons",
+      "title": "ACCOuNT Data Commons (QA)",
```

Comment:
No.

Reply:
Oh, I see the problem here 😓 , let me prepare a fix so the automation will not replace that value in the next run.